internals/daemon/api_notices.go

```diff
@@ -39,9 +41,23 @@ type addedNotice struct {
 	ID string `json:"id"`
 }
 
+var (
+	errUserIDFilterNoNotices = errors.New("no notices possible with user IDs filter for given request")
+)
+
 func v1GetNotices(c *Command, r *http.Request, _ *UserState) Response {
 	query := r.URL.Query()
 
+	reqUid := uidFromRequest(r)
+
+	userIDs, err := sanitizeUserIDsFilter(reqUid, query["user-ids"])
+	if err == errUserIDFilterNoNotices {
```

Comment:
These days it's best to use [`errors.Is`](https://pkg.go.dev/errors#Is) in case the function has wrapped the error:

```go
if errors.Is(err, errUserIDFilterNoNotices) {
```